stable/grafana/templates/configmap.yaml

```diff
@@ -48,14 +48,21 @@ data:
   {{- range $provider, $dashboards := .Values.dashboards }}
     {{- range $key, $value := $dashboards }}
       {{- if (or (hasKey $value "gnetId") (hasKey $value "url")) }}
-    curl -sk \
-    --connect-timeout 60 \
-    --max-time 60 \
-    -H "Accept: application/json" \
-    -H "Content-Type: application/json;charset=UTF-8" \
+    wget -q --no-check-certificate \
```

Comment:
Switching from curl to wget is unrelated to the stated purpose of this pull request.

Reply:
alpine/git image has wget instead off curl